cumulus_library/template_sql/statistics/count.sql.jinja

```diff
@@ -66,16 +96,22 @@ CREATE TABLE {{ table_name }} AS (
             count(DISTINCT {{ secondary }}_ref) AS cnt_{{ secondary }},
             {%- endif -%}
             {% for col in table_cols %}
-            "{{ col }}"
+            "{{ column_or_alias(col) }}"
             {{- comma_delineate(loop) }}
             {%- endfor %}
+            {%- if fhir_resource=='condition' -%},
+            cond_code_display
+            {% endif %}
         FROM null_replacement
         GROUP BY
-            cube( 
+            cube(
                 {%- for col in table_cols %}
-                "{{ col }}"
+                "{{ column_or_alias(col) }}"
                 {{- comma_delineate(loop) }}
                 {%- endfor %}
+                {%- if fhir_resource=='condition' -%},
+                cond_code_display
+                {% endif %}
```

Comment:
Yo there's a lot of condition specific code now :smile:  - This is stuff that was missing before and Condition is just a pain in the butt, I'm gathering, rather than the result of all your schema work.

Reply:
I :think: that it still makes sense to keep all these running through the same template since 80% is boilerplate, but i think there's going to be more of this kind of thing as we get the other count types out of static sql and into builders, though at this point i think it should mostly be just elif statements against these blocks.